deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -80,93 +102,101 @@ spec:
       - kind: Services
         name: ""
         version: v1
+      - kind: ServiceMonitors
+        name: servicemonitors.monitoring.coreos.com
+        version: v1
       specDescriptors:
       - description: Enable Prometheus for metrics backend.
         displayName: Prometheus enabled
         path: backends.metrics.prometheus.enabled
         x-descriptors:
-        - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:Backends'
+        - urn:alm:descriptor:com.tectonic.ui:fieldGroup:Backends
```

Comment:
Updating the CSV via `operator-sdk generate bundle` resulted in some clean up to the CSV with the 0.19.4 version of operator-sdk, so you'll see a bunch of these changes which were not made by me manually :)